README.md

```diff
@@ -2,73 +2,250 @@
 
 The best way to handle your search bar documentation with [MeiliSearch](https://github.com/meilisearch/MeiliSearch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+The goal of this library is to provide very easy integration of a search bar into your own documentation. To make that possible, you need to scrape your website content in advance, and index it in a MeiliSearch instance.
```

Comment:
Scraiping is not the only way to gather your website content. Most multi-language websites have their content already stored in databases or files and thus they do not need to be parsed.

```suggestion
The goal of this library is to provide very easy integration of a search bar into your own documentation. To make that possible, you need to gather your website content in advance, and index it in a MeiliSearch instance.
```